.github/workflows/eks-ci.yml

```diff
@@ -8,6 +8,11 @@ on:
         required: true
         description: "Specify test markers to run"
         default: '["eks"]'
+  
+  # On post-merge execution running eks tests
+  push:
+    branches:
+      - main
```

Comment:
do you need to add the inputs like the in the `workflow_dispatch` flow?

Reply:
For post-merge - no